autoscaler/controllers/gateway/config/elasticsearch_test.go

```diff
@@ -0,0 +1,50 @@
+package config_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/keyval-dev/odigos/autoscaler/controllers/gateway/config"
+	"github.com/stretchr/testify/require"
+)
+
+func TestElasticsearch_SanitizeURL(t *testing.T) {
```

Comment:
I suggest adding a test for failure. e.g. that the `SanitizeURL` function returns an error for invalid URL endpoint

Reply:
this is done ✅